python/nav/web/seeddb/urls.py

```diff
@@ -118,29 +118,25 @@
     path('service/bulk/', service.service_bulk, name='seeddb-service-bulk'),
     # Room
     path('room/', room.room, name='seeddb-room'),
-    re_path(r'^room/edit/(?P<room_id>.+)/$', room.room_edit, name='seeddb-room-edit'),
-    re_path(
-        r'^room/delete/(?P<object_id>.+)/$', room.room_delete, name='seeddb-room-delete'
-    ),
+    path('room/edit/<room_id>/', room.room_edit, name='seeddb-room-edit'),
```

Comment:
Nitpick, but it would be nice to have type annotation for path parameters (e.g. `int:room_id`) in this file